doc/kernel/data_structures/index.rst

```diff
@@ -20,10 +20,12 @@ behind this design is to allow the collections to be used in contexts
 where dynamic allocation is disallowed (i.e. there is no need to
 allocate node objects because the memory is provided by the user).
 
-Note also that these libraries are uniformly unsynchronized; access to
+Note also that these libraries are generally unsynchronized; access to
```

Comment:
Wasn't this already merged via https://github.com/zephyrproject-rtos/zephyr/pull/79943?